rtsf-at-checkout-event-simulator/main.go

```diff
@@ -156,39 +138,22 @@ func (chkoutEvt *CheckoutEvent) send(eventTime time.Time) (waitTime time.Duratio
 		os.Exit(-1)
 	}
 
-	if chkoutEvt.Device == rspName {
```

Comment:
Based On Lenny's comments below I believe we would handle the rfid-roi-event with this logic.

Reply:
NOTE: remove for now and reevaluate when we look to add LLRP